Pilot/vm-pilot.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
```

Comment:
Contact Andrew, but IIUC this file: https://gitlab.cern.ch/mcnab/temp-diracpilot/blob/master/user_data is the "new" vm-pilot" file, or at least most things converged there.